DB/src/main/java/io/deephaven/db/v2/utils/UpdatePerformanceTracker.java

```diff
@@ -362,9 +366,7 @@ public final void onUpdateEnd() {
             startCpuNanos = 0;
             startTimeNanos = 0;
 
-            final RuntimeMemory runtimeMemory = RuntimeMemory.getInstance();
-            totalFreeMemory = runtimeMemory.freeMemory();
-            totalUsedMemory = runtimeMemory.totalMemory();
+            RuntimeMemory.getInstance().read(endMemSample);
```

Comment:
Put this before we start zeroing things out. Arguably it should be the first thing in the method.